tools/src/test/java/org/apache/kafka/tools/GetOffsetShellTest.java

```diff
@@ -283,7 +318,7 @@ public void testTopicPartitionsArgWithInternalExcluded() {
         assertEquals(expected, offsets);
     }
 
-    @ClusterTest
+    @ClusterTest(clusterType = Type.ZK)
```

Comment:
Could we create consumer to create internal topic for this test case?

Reply:
Sure - amended the test to create consumer for polling